code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2472,6 +2472,51 @@
 		T.health -= 20
 		T.mutation_mod += 0.1
 
+/datum/reagent/toxin/insecticide
+	name = "Insecticide"
+	id = INSECTICIDE
+	description = "A broad pesticide. Do not ingest!"
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#49002E" //rgb: 73, 0, 46
+	density = 1.08
+	specheatcap = 4.18
+
+/datum/reagent/toxin/insecticide/reaction_mob(var/mob/living/M, var/method = TOUCH, var/volume)
+
+	if(..())
+		return 1
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if(!C.wear_mask) //If not wearing a mask
+			C.adjustToxLoss(REM) //4 toxic damage per application, doubled for some reason
+		if(ishuman(M) || ismonkey(M))
+			var/mob/living/carbon/human/H = M
+			if(H.dna)
+				if(H.species.flags & isinsectoid(M)) //Insecticide being poisonous to bugmen, who'd've thunk
+					H.adjustToxLoss(10 * REM)
+			if(H.species.flags & isroach(M))
```

Comment:
isroach doesn't exist and you already gib animals, remove this check

Reply:
Nevermind, I think you meant to damage the primitive forms of insectoids too
Replace this check with "if(istype(M, /mob/living/carbon/monkey/roach))"